### PR TITLE
Email services crud operations

### PR DESCRIPTION
**Pull request by v-vprasannak**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

**Comment by v-vprasannak** [comment]
@microsoft-github-policy-service agree company="Microsoft"

File: .github/policies/cla.yml [hunk 1]
```diff
@@ -0,0 +1,130 @@
+name: Contributor License Agreement Policy
+description: CLA policy file
+
+resource: repository
```
Comment (kairu-ms) [inline .github/policies/cla.yml:4]: What's this file used for?

File: .github/policies/platformcontext.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+name: platform_context
+description: The context for GitOps platform, this will drive GitOps specific policies
+owner:
```
Comment (kairu-ms) [inline .github/policies/platformcontext.yml:3]: Please remove this file.

File: src/communication/azext_communication/aaz/latest/emailcommunication/__cmd_group.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command_group(
+    "communication email",
+)
```
Comment (kairu-ms) [inline src/communication/azext_communication/aaz/latest/emailcommunication/__cmd_group.py:16]: Why the generated folder name is NOT `communication\email`? Please do not manually change the folde of `aaz`, it should be directly generated from `aaz-dev-tools`.

File: src/communication/azext_communication/manual/_help.py [hunk 4]
```diff
@@ -575,3 +575,165 @@
...
 """
+
+helps['communication email create'] = """
+    type: command
+    short-summary: "Creates a new Email Communication Service or update an existing one."
+    examples:
+      - name: Create or update email communication resource
+        text: |-
+               az communication email create --name "MyEmailCommunicationResource" --resource-group "MyResourceGroup" --location "Global" --data-location "UnitedStates"
+"""
```
Comment (kairu-ms) [inline src/communication/azext_communication/manual/_help.py:586]: The help message is not required adding in `_help.py` for the `aaz` commands. They are already contained in the the aaz commands.

**Pull request closed without merging**